module/cmd/gravity/cmd/root.go

```diff
@@ -114,6 +114,10 @@ func Execute(rootCmd *cobra.Command) error {
 }
 
 func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
+
+	cfg := sdk.GetConfig()
+	cfg.Seal()
```

Comment:
digging into how this works it just seems strange. Pointer magic going on about sealing the config globally using a reference to it.

I can understand how this pattern would cause mistakes, shouldn't the way this works be modified in the sdk to avoid 'spooky action at a distance' which seems to be the root cause of this bug. 

Anyways merging for now.